src/order/complete_lattice.lean

```diff
@@ -39,7 +39,7 @@ class has_Sup (α : Type*) := (Sup : set α → α)
 /-- class for the `Inf` operator -/
 class has_Inf (α : Type*) := (Inf : set α → α)
 
-export has_Sup (Sup) has_Inf (Inf)
+open has_Sup (Sup) has_Inf (Inf)
```

Comment:
What is this for? This looks disruptive to other parts of the library.

Reply:
Otherwise, I can't use `complete_partial_order.Sup` without qualifying it.

Reply:
Can you maybe not call it `Sup`?

Reply:
What would you call it?

There is one way we could unify with `has_Sup`: we'd need to be ok with giving `has_Sup` an additional parameter:

```lean
class has_Sup (a b : Type*) :=
(Sup : a -> b)
```

Reply:
No, that would make type inference impossible. How about something like `\omegaSup`?